input/fsh/actors.fsh

```diff
@@ -31,3 +30,14 @@ Usage: #definition
 * description = "The AU eRequesting Patient Access is a system that is the digital interface that allows patients or their representatives to view and manage requests for diagnostic services. It provides access to request information, enabling patients to choose service providers and track the status of requests."
 * type = #system
 * documentation = "An AU eRequesting Patient Access (client): TBD"
+
+Instance: au-erequesting-actor-server
+InstanceOf: ActorDefinition
+Usage: #definition
+* url = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-server"
+* name = "AUeRequestingServer"
+* title = "AU eRequesting Server"
+* status = #draft
+* description = "The eRequesting Server system accepts diagnostic service requests and makes diagnostic service requests accessible."
```

Comment:
Change the definition to be consistent with the others. The AU eRequesting Server is a system that ...". Proliferate this change as needed.

Reply:
Changed AU eRequesting Server actor definition. Updated in index and actors pages.